wled00/data/settings_leds.htm

```diff
@@ -422,7 +422,7 @@
 <option value="15">15mA (seed/fairy pixels)</option>
 <option value="0">Custom</option>
 </select><br>
-<div id="LAdis${s}" style="display: none;">max. mA/LED: <input name="LA${s}" type="number" min="1" max="255" oninput="UI()"> mA<br></div>
+<div id="LAdis${s}" style="display: none;">max. mA/LED: <input name="LA${s}" type="number" min="1" max="255" oninput="UI()" value=55> mA<br></div>
```

Comment:
might be better to put the value in quotes? like `value="55"`

it might also be possible to use the JS var `laprev`

https://github.com/Aircoookie/WLED/blob/99427c2ef734eb0bd04ec2c5f9d21655d0349e25/wled00/data/settings_leds.htm#L9

Reply:
> might be better to put the value in quotes? like value="55"

I should've looked that up; I'd assumed a numeric field required a numeric value, but you're right - it should be quoted.

> it might also be possible to use the JS var laprev

Oooh, good catch!  This variable is never used - perhaps the intent was to make the next new bus inherit the values from the previous one, which is a reasonable default.  I'm inclined to remove it for now - that's beyond my level of JS skill, especially given the upcoming release - but leave an issue entry for the duplication behaviour for the future.